packages/web/src/components/VaultCard/VaultCard.tsx

```diff
@@ -366,9 +426,11 @@ export const VaultCard = ({
           <div className="assets">
             <span className="subtitle">{auditPayout ? t("paidAssets") : t("assetsInVault")}</span>
             <VaultAssetsPillsList auditPayout={auditPayout} vaultData={vaultData} />
+
+            <OptedInList editSessionIdOrAddress={vault.id} />
           </div>
           <div className="actions">
-            {(!isAudit || (isAudit && vault.dateStatus !== "finished" && !auditPayout)) && (
+            {!isAudit && (
```

Comment:
why was the old condition removed?

Reply:
because showing the "deposits" button makes no sense for competitions even if they are finished